package.json

```
@@ -413,27 +413,14 @@
                 },
                 "default": {}
               },
-              "dataEditor.omegaEditPort": {
-                "type": "integer",
-                "description": "Editor server default port",
-                "default": 9000
-              },
-              "dataEditor.logFile": {
-                "type": "string",
-                "description": "Path to log file for data editor",
-                "default": "${workspaceFolder}/dataEditor-${omegaEditPort}.log"
-              },
-              "dataEditor.logLevel": {
-                "type": "string",
-                "description": "Log level for data editor",
-                "enum": [
-                  "error",
-                  "warn",
-                  "info",
-                  "debug",
-                  "trace"
-                ],
-                "default": "info"
+              "dataEditorConfig": {
```

```suggestion
              "dataEditor": {
```
`Config` seems redundant.

---

Agreed.

---

@arosien Should this be changed everywhere `dataEditorConfig` is used or just inside of package.json? Meaning like the interface that was created, etc.

---

I just renamed the package.json settings. So that in TS code you still know its a dataEditorConfig not the actual dataEditor.